gui/default/index.html

```diff
@@ -208,11 +208,14 @@ <h3 class="panel-title">
                 </p>
               </div>
               <div class="panel-footer clearfix">
+                <button type="button" class="btn btn-sm btn-default pull-left" ng-click="dismissPendingDevice(deviceID)" tooltip data-original-title="{{'Do not add it to the ignore list, so this notification may recurr.' | translate}}">
+                  <span class="far fa-clock"></span>&nbsp;<span translate>Not Now</span>
```

Comment:
I tend to prefer "Dismiss" over "Not Now" - to me the latter implicates it will come up again (e.g. just hidden until refresh/some time passed). While in reality it might come up again, but it's not guaranteed.

Reply:
or "Later"

Reply:
Changed to "Dismiss".